wled00/ir.cpp

```diff
@@ -654,9 +654,7 @@ void handleIR()
       {
         if (results.value != 0) // only print results if anything is received ( != 0 )
         {
-          Serial.print("IR recv\r\n0x");
-          Serial.println((uint32_t)results.value, HEX);
-          Serial.println();
+          Serial.printf("IR recv: 0x%X\n", (uint32_t)results.value);
```

Comment:
printf uses precious RAM. (As unfortunately does original code.)
Please use F() macro to utilize flash memory.

Reply:
Ah, will change to `F()` macro.

Looking at the code it looks to me as long as the message is <64 bytes, `print` and `printf` use roughly the same amount of stack and no dynamic allocation.  (print allocates enough to encode a long as binary...  `8x8 + 1`.)